### PR TITLE
observerevent event type issue fixed

### PR DESCRIPTION
**Pull request by RavindraChherke**
Looks like the MAKE_SYSTEM_PROP was not setting event values properly. I think keeping it in Module file is a good idea. Also, DATA_EVENT_TYPE_VALUE was set to FIRDataEventTypeChildAdded. Now changed to FIRDataEventTypeValue.

### CODE REVIEW

File: ios/Classes/FirebaseDatabaseModule.m [hunk 1]
```diff
@@ -16,6 +16,12 @@ @implementation FirebaseDatabaseModule
 
 #pragma mark Internal
 
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_VALUE, FIRDataEventTypeValue);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_ADD, FIRDataEventTypeChildAdded);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_REMOVE, FIRDataEventTypeChildRemoved);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_MOVE, FIRDataEventTypeChildMoved);
+MAKE_SYSTEM_PROP(DATA_EVENT_TYPE_CHANGE, FIRDataEventTypeChildChanged);
```
Comment (hansemannn) [inline ios/Classes/FirebaseDatabaseModule.m]: Looks good! One typo I noticed (that was in my initial implementation as well) is that we should include the `CHILD` phrase as well, e.g.:
- `DATA_EVENT_TYPE_CHILD_ADDED`
- `DATA_EVENT_TYPE_CHILD_REMOVED`
- `DATA_EVENT_TYPE_CHILD_MOVED`
- `DATA_EVENT_TYPE_CHILD_CHANGED`

Can you change that in code and docs?

**Review by hansemannn — CHANGES_REQUESTED** [review]
Left one review comment. Also, please bump the version in ios/manifest to the next patch version.

**Comment by RavindraChherke** [comment]
I have made the changes as suggested. Please check. Thanks!

**Comment by hansemannn** [comment]
It should be a patch bump (1.1.1) instead of a minor bump (1.2.0), referring to the [semver specs](https://semver.org/). Also, the constants should be suffixed with `_CHANGED` instead of `_CHANGE`. Please update that last piece and attach the new module version. Thx!

**Comment by RavindraChherke** [comment]
Sorry, I should pay more attention to details. This is my first time contributing to other project.

I have build the module with Titanium SDK 7.1.1  There are changes in titanium.xcconfig. Is that OK?

Please find the attached zip file. 

[firebase.database-iphone-1.1.1.zip](https://github.com/hansemannn/titanium-firebase-database/files/2034133/firebase.database-iphone-1.1.1.zip)

**Comment by hansemannn** [comment]
Looks perfect! PR **approved**. Keep up the great contributions to the community.

**Review by hansemannn — APPROVED** [review]

**Comment by hansemannn** [comment]
New Release: https://github.com/hansemannn/titanium-firebase-database/releases/tag/ios-1.1.1

**Pull request merged**